src/avgScrabbleScore.js

```diff
@@ -1,20 +1,28 @@
 let scrabbleScore = require('./scrabbleScore');
-
+["cat", "dog", "snake"]
```

Comment:
Remove this before submitting for feedback. Use `git diff` to see what changes you've made before using `git add` to create commits.